documentation/Readme.md

```diff
@@ -0,0 +1,51 @@
+# Documentation
+
+## Introduction
+
+`Node-MySQL2` is aims to be a drop in replacement for [node-mysql](https://github.com/felixge/node-mysql). Please check `node-mysql` for full documentation.
```

Comment:
'is aims' - is that correct grammar?

also maybe change links from node-mysql to mysqljs org?